torch/csrc/jit/passes/quantization.cpp

```diff
@@ -1857,7 +1862,7 @@ ModuleMethodVector InsertQuantDeQuantHelper::getInvokedMethods(
         if (module_instance == graph->inputs()[0]) {
           m = module;
         } else {
-          m = findChildModuleToQuantize(module, module_instance);
+          m = findChildModuleToQuantize(module, n, graph->inputs()[0]);
```

Comment:
I think you can remove this function and just copy the function body to here

Reply:
Sure